main.py

```diff
@@ -45,13 +40,22 @@ def run_predict_model():
     parser.add_argument("--train_model", type=bool)
     parser.add_argument("--predict_model", type=bool)
     args = parser.parse_args()
+
+    dataset_maker = DatasetMaker(
+        input_file_name="final_taxi_data",
+        test_size=0.3,
+        save_files=True,
+        output_train_test_file_names=("train_data", "test_data"),
+        output_cleaned_file_name="train_data_clean",
```

Comment:
I thought we agreed, to choose dict's one key into each class. I would expect here sth like params['dataset_params']

Reply:
I thought to leave it as it is until we have a config file. But, you're right - I will create temporary dict in main.py to store all the parameters for now

Reply:
so comes a question if config to be done now, I believe so. But it adds bigger chunk of code in one PR